src/simulation/LoadGenerator.h

```diff
@@ -67,6 +119,60 @@ struct GeneratedLoadConfig
     bool skipLowFeeTxs = false;
     // Percentage (from 0 to 100) of DEX transactions
     uint32_t dexTxPercent = 0;
+
+    // The following ranges are only used for SOROBAN_INVOKE txs:
```

Comment:
I think the list of the parameters and possible combinations is growing unwieldy. For the sake of time we probably can keep this as is for now, but as a followup it would be nice to introduce a few structs that group the related parameters together (invoke parameters, soroban upgrades, soroban setup parameters, maybe something else).

Reply:
Agreed, loadgen has increased in complexity quite a bit. We should probably open an issue so we don't forget to clean this up. It also wouldn't hurt to make sure we don't allow invalid combinations of parameters (e.g. it doesn't make sense to accept txs/accounts/etc in soroban upgrade mode)